### PR TITLE
EWSv2

### PR DESCRIPTION
**Pull request by dantavori**
## Status
Ready

## Related Issues
fixes: [https://github.com/demisto/etc/issues/18380](https://github.com/demisto/etc/issues/18380)

## Description
removed unnecessary try except in move_item_between_mailboxes function 

## Does it break backward compatibility?
   - No

### CODE REVIEW

File: Integrations/EWSv2/EWSv2.yml [hunk 1]
```diff
@@ -60,8 +60,8 @@ configuration:
   name: defaultServerVersion
   required: false
   type: 0
-- defaultvalue: 'true'
-  display: Trust any certificate (unsecure)
+- defaultvalue: ''
```
Comment (IkaDemisto) [inline Integrations/EWSv2/EWSv2.yml]: Changing the default value to false can cause ssl verification error at users that already configured `EWS v2`, so it should be taken into consideration.

Comment (dantavori) [reply Integrations/EWSv2/EWSv2.yml]: After an instance's parameter value was set (with default value or not), changing the parameter's default value in the .yml file won't affect the current value.

**Comment by kirbles19** [comment]
@dantavori Done.

**Comment by Itay4** [comment]
@dantavori ?

**Comment by jpeng19** [comment]
@jpeng19 
https://demisto.zendesk.com/agent/tickets/6983

**Pull request merged**